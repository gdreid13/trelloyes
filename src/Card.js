import React from 'react';
import Card from 'Card.css';

function Card(props) {
    <div className="card">
        <h3>{props.title}</h3>
        <p>{props.content}</p>
    </div>
}

export default Card;