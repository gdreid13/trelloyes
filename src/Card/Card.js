import React from 'react';
import card from '../Card/card.css';

function Card(props) {
    <div className="Card">
        <h3>{props.title}</h3>
        <p>{props.content}</p>
    </div>
}

export default Card;