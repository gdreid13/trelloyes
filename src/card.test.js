import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer'
import Card from './Card';

it ('snapshot of card <Card />', () => {
  const snap = renderer.create(<Card props={[]}/>).toJSON();
  expect(snap).toMatchSnapshot();
});