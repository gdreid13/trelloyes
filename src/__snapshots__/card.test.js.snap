// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card component snapshot of card 1`] = `
<div
  className="Card"
>
  <h3 />
  <p />
</div>
`;
